sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Customize/ArmDesktopVirtualizationModelFactory.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.DesktopVirtualization.Models;
+using Azure.ResourceManager.Models;
+using SystemData = Azure.ResourceManager.Models.SystemData;
+using Azure.ResourceManager.DesktopVirtualization;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Customize
```

Comment:
this namespace should be same as the generated `ArmDesktopVirtualizationModelFactory` class
```suggestion
namespace Azure.ResourceManager.DesktopVirtualization.Models
```